content/2.0.html.haml

```diff
@@ -34,12 +34,11 @@ title: "Jenkins 2.0"
 
 
 %h1
-  Jenkins 2.0 preview
+  Jenkins 2.0 Overview
 
 %p
-  %strong
-    Jenkins 2.0
-  is an upcoming major version of Jenkins with a number of usability enhancements and new features.
+  Jenkins 2.0 is an automation engine with an unparalleled plugin ecosystem to support all of your favorite tools in your delivery pipelines, whether your goal is continuous integration, automated testing or continuous delivery. 
```

Comment:
I hear you but keep in mind that a lot of non-techie (read journalists) etc will come to this page for a sound bit. I would like to get them to hear the elevator pitch right away. The previous description was very weak. So I recommend that we keep this here.